### DISCUSSION TITLE
Visual Studio 2019 - "Unable to resolve the .NET SDK version as specified in the global.json located at..."

### DISCUSSION THREAD

**Discussion by benconnolly-python** [mozts2005/ZendeskApi_v2#504] [Q&A] [answered]
I am interested in your Zendesk_API_V2. I'm looking to get some assistance in getting the ZendeskApi_v2.Example working.
I'm running into Build issues using Visual Studio 2019. Specifically, I'm getting an error indicating "Unable to resolve the .NET SDK version as specified in the global.json located at..." - I downloaded using the ZIP file

![2021-02-11_14-50-40](https://user-images.githubusercontent.com/69989404/107690859-ad0b3400-6c78-11eb-9a6c-048448fd08c1.jpg)

**Comment by mozts2005** [comment, marked as answer]
@benconnolly-python the global.json used in this project references the .NET SDK version you need to have installed. 

https://github.com/Speedygeek/ZendeskApi_v2/blob/c3998db6710b8ed90dc81eee0661d1b8eecb5b3b/src/global.json#L3

Please install the correct version: https://dotnet.microsoft.com/download/dotnet-core/thank-you/sdk-3.0.103-windows-x64-installer and restart your system.

**Comment by jecoute** [comment]
I can build ZendeskApi_v2.csproj ok.
I can build ZendeskApi_v2.Example.csproj ok.
I cannot build ZendeskApi_v2.Tests.csproj.

I'm using Visual Studio 2019.    To get around the issue, I've excluded if from the solution build.  Can someone help?   

![image](https://user-images.githubusercontent.com/1268162/136641746-c3bba624-5c72-4703-8bfa-7437a1b989aa.png)

![image](https://user-images.githubusercontent.com/1268162/136641824-c6d03ba1-833c-42bb-85f7-02cc516d9e46.png)

**Reply by mozts2005** [reply]
please, make sure you have the 3.1 LTS SDK installed.

**Reply by jecoute** [reply]
Thanks I did install it and when I try to build that project now, it just hangs.   Never finishes the build.   I love the project though.  We've started using zendesk at my employer, and I'm using the nuget package and it is great.  There are a few things missing and that's why I came here to help contribute to the project.  :)

**Comment by jecoute** [comment]
I actually dropped the folder and took the files from the zip for a clean version again, but still getting some errors.   I will try again tomorrow.

![image](https://user-images.githubusercontent.com/1268162/136643290-a7315f70-c8d9-46ba-9178-78411139a6eb.png)
